torchmetrics/functional/classification/auroc.py

```diff
@@ -99,7 +99,7 @@ def _auroc_compute(
 
             # calculate average
             if average == AverageMethod.NONE:
-                return auc_scores
+                return tensor(auc_scores)
```

Comment:
how did you find this issue if it's not covered by tests?

Reply:
mypy :]